### PR TITLE
batch changes

### PR DESCRIPTION
**Pull request by mrogowski**

### CODE REVIEW

**Comment by mrogowski** [comment]
This can be reviewed for now, might create another PR later. @stefanozampini @dalcinl

**Comment by stefanozampini** [comment]
If you plan to fix the multiline summary later, it is fine for me.

**Review by stefanozampini — APPROVED** [review]

File: src/binding/petsc4py/src/petsc4py/PETSc/DMPlex.pyx [hunk 1]
```diff
@@ -166,7 +167,9 @@ cdef class DMPlex(DM):
 
         See Also
         --------
-        DM, DMPlex, DM.setFromOptions, DMPlex.createBoxMesh, DMPlex.createFromFile, DM.setType`, DM.create, petsc.DMPlexCreateBoxSurfaceMesh
+        DM, DMPlex, DM.setFromOptions, DMPlex.createBoxMesh
+        DMPlex.createFromFile, DM.setType`, DM.create
```
Comment (mrogowski) [inline src/binding/petsc4py/src/petsc4py/PETSc/DMPlex.pyx:171]:
```suggestion
        DMPlex.createFromFile, DM.setType, DM.create
```

File: src/binding/petsc4py/src/petsc4py/PETSc/SF.pyx [hunk 2]
```diff
@@ -415,10 +415,10 @@ cdef class SF(Object):
     def compose(self, SF sf) -> SF:
         """Compose a new `SF`.
 
-        Puts the *sf* under this object in a top (roots) down (leaves) view.
-
         Collective.
 
+        Puts the *sf* under this object in a top (roots) down (leaves) view.
```
Comment (mrogowski) [inline src/binding/petsc4py/src/petsc4py/PETSc/SF.pyx:420]:
```suggestion
        Puts the ``sf`` under this object in a top (roots) down (leaves) view.
```

**Pull request merged**